<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsuarioRegisError</class>
 <widget class="QDialog" name="UsuarioRegisError">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="botonRegresaLogin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Error, intentalo de nuevo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
